docs/testplan/GCU-Dynamic-ACL-testplan.md

```diff
@@ -137,8 +137,66 @@ Verify that after creation, a forward ACL rule can be removed and packets matchi
 - Verify that the results of "show acl rule {rule_name}" are empty for both rule names
 
 - Verify that packets with IPs matching the removed forwarding rules are dropped on this specific port
+-
+### Test Case # 7 - DHCP Forwarding Rule
 
-### Test Case # 7 - Scale test of ACL Table, add large amount of forward and drop rules
+#### Test Objective
+
+Verify that GCU is capable of adding blanket DHCP forwarding rules, and that traffic follows these rules accordingly
+
+#### Testing Steps
+
+- Use GCU to create blanket DHCP packet forwarding rule
+
+- Use GCU to create a blanket drop rule on a port
+
+- Verify that for both rules created, "show acl rule DYNAMIC_ACL_TABLE {rule_name} matches expected output
+
+- Verify that all operations were successful
+
+- Run pre-existing DHCP_Relay test with ACL rules applied and packets sent over the port that is blocked
```

Comment:
What's the pre-existing DHCP_Relay test? And should the DHCP packet being forwarded rather than dropped?

Reply:
The preexisting test is found [here](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/dhcp_relay/test_dhcp_relay.py). With the DHCP forwarding rule applied, we would expect to see DHCP packets still forwarded and an entire relaying sequence be completed despite being sent over a port blocked by the lower priority drop rule

Reply:
@jongorel , how would you run pre-existing DHCP_Relay test, would you invoke test_dhcp_relay.py in your test script?

Reply:
@yaqiangz under this approach I would copy the "test_dhcp_relay_default" methods & associated fixtures into their own modules named test_dhcp_relay_acl.py and test_dhcpv6_relay_acl.py, and apply the associated custom ACLs before invoking the ptf_runner DHCP/DHCPv6 test, then tear it down after.

Reply:
I see, as @bingwang-ms said, suggest making this part of test plan clearer like your reply. It's a bit confusing.